src/Toggle/Toggle.test.js

```diff
@@ -53,6 +54,32 @@ describe('<Toggle />', () => {
         expect(wrapper.state(['checked'])).toBeTruthy();
     });
 
+    describe('Toggle default rendering', () => {
+        test('should default to a not checked state', () => {
+            const element = mount(<Toggle />);
+
+            expect(element.state(['checked'])).toBe(false);
+        });
+        test('should have truthy checked state when passed checked prop', () => {
+            const element = mount(<Toggle checked />);
+
+            expect(element.state(['checked'])).toBe(true);
+        });
+    });
+
+    describe('onChange handler', () => {
+        test('should dispatch the onChange callback with the event', () => {
+            let spy = sinon.spy();
```

Comment:
If we don't want to add sinon as a dependency jest has a built in mock function: `jest.fn()` https://jestjs.io/docs/en/mock-functions